code/controllers/AssetAdmin.php

```diff
@@ -62,13 +58,11 @@ public function currentPageID() {
 	}
 
 	/**
-	 * Set up the controller, in particular, re-sync the File database with the assets folder./
+	 * Set up the controller
 	 */
 	public function init() {
 		parent::init();
 
-		// Create base folder if it doesnt exist already
-		if(!file_exists(ASSETS_PATH)) Filesystem::makeFolder(ASSETS_PATH);
```

Comment:
Confirmed that flysystem creates `ASSETS_PATH` automatically: https://github.com/thephpleague/flysystem/blob/master/src/Adapter/Local.php#L73